samples/pom.xml

```diff
@@ -34,6 +34,11 @@
 
   <name>Apache Apex Malhar (incubating) Samples</name>
 
+  <properties>
+    <semver.plugin.skip>true</semver.plugin.skip>
```

Comment:
I guess not, but this is how I found it in another module. However, why would be require a per module plugin configuration for something that by default should apply to all modules?